packages/libs/react-ui/src/components/NavFooter/FooterLink.tsx

```diff
@@ -14,11 +14,16 @@ export const FooterLink: FC<IFooterLinkProps> = ({ children }) => {
   };
 
   const clones = React.Children.map(children, (child) => {
-    // @ts-ignore
```

Comment:
Unrelated to PR. Just something I noticed and seemed small enough to push through.